javascript/javascript2/week1/index.js

```diff
@@ -14,17 +14,18 @@ console.log(danishWords); // returns 'ø'
 const danishString = "Jeg har en blå bil";
 
 function notThisFunctionName(params) {
-  let letterA = params.split(" ").filter((e) => e === "å");
-  return console.log(`total: ${letterA.length + 0} å:${letterA.length} `);
+  let letterA = params.split(" ").filter((e) => e.includes("å"));
```

Comment:
so cool that you are improving your old homework with the new knowledge !👨🏻‍🎨

Reply:
Fixed